website/docs/r/media_streaming_policy.html.markdown

```diff
@@ -0,0 +1,223 @@
+---
+subcategory: "Media"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_media_streaming_policy"
+description: |-
+  Manages a Streaming Policy.
+---
+
+# azurerm_media_streaming_policy
+
+Manages a Streaming Policy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "media-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestoracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_media_services_account" "example" {
+  name                = "examplemediaacc"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id         = azurerm_storage_account.example.id
+    is_primary = true
+  }
+}
+
+resource "azurerm_media_streaming_policy" "example" {
+  name                        = "Policy-1"
+  resource_group_name         = azurerm_resource_group.example.name
+  media_services_account_name = azurerm_media_services_account.example.name
+  no_encryption_enabled_protocols {
+    download         = true
+    dash             = true
+    hls              = true
+    smooth_streaming = true
+  }
+}
+```
+
+## Example Usage with Secure Streaming
```

Comment:
could we remove the other example and make this the default (since it's more feature-complete?)

```suggestion
## Example Usage
```